rules/consistent-test-it.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const { getNodeName, isTestCase, isDescribe } = require('./util');
+
+module.exports = {
+  meta: {
+    docs: {
+      url:
+        'https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/consistent-test-it.md',
+    },
+    fixable: 'code',
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          fn: {
+            enum: ['it', 'test'],
+          },
+          withinDescribe: {
+            enum: ['it', 'test'],
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
+  },
+  create(context) {
+    const configObj = context.options[0];
```

Comment:
I think you could declare this like:

```js
const configObj = context.options[0] || {}
```

And then you're always dealing with a truthy object, so that on the following lines, you do not need to call `configObj && configObj.propertyName`.

Reply:
I agree about that. Thanks